src/LayerEditor/ui/panels/regions_panel.py

```diff
@@ -160,11 +149,11 @@ class Regions(QtWidgets.QToolBox):
     pyqtSignals:
         * :py:attr:`regionChanged() <regionChanged>`
     """
-    color_changed = QtCore.pyqtSignal()
+    regions_changed = QtCore.pyqtSignal()
 
     # Emitted when color of some region has changed.
 
-    def __init__(self, layer_heads, parent=None):
+    def __init__(self, le_data, parent=None):
```

Comment:
Rather pass only the block, ultimately replace BlockLayerHeads.

Reply:
Done. Replaced by LEModel.